discussion/templates/discussion/discussion.html

```diff
@@ -135,7 +135,7 @@
           </div>
           <div id="discussion-your-comment-tips">
 
-            <label for="discussion-attach-file"><span class="glyphicon glyphicon-paperclip" style="margin-right: 3px"></span>Attach a File</label>
+            <label for="discussion-attach-file"><span class="glyphicon glyphicon-paperclip" style="margin-right: 3px"></span> <span href="#" title="Accepted file types are PDF(application/pdf), PNG(image/png), and JPG(image/jpg)">Attach a File</span></label>
```

Comment:
Needs spaces before open parens, e.g.: `PDF (application/pdf)`.

Make sure to use valid [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types), so use `image/jpeg` rather than `image/jpg`.